Services/OnScreenChat/classes/DTO/MessageDto.php

```diff
@@ -24,7 +24,7 @@
  */
 class MessageDto
 {
-    private string $id;
+    private string $id; //TODO PHP8-REVIEW: missing property type. getId() returns string
```

Comment:
@Uwe-Kohnle Hmmm, there is already a typed property in the `trunk`.

IMO I can close this PR because there is no TODO left I could tackle :-).